package.json

```diff
@@ -35,6 +35,7 @@
   "dependencies": {
     "adm-zip": "^0.4.13",
     "archiver": "^1.1.0",
+    "ask-smapi-model": "1.6.0",
```

Comment:
this will allow to control version of ask-smapi-model explicitly. we dont need any whitelist then.